sns_ik_examples/config/example_sawyer_joint_limits.yaml

```diff
@@ -0,0 +1,36 @@
+joint_limits:
+  right_j0:
+    has_velocity_limits: true
+    max_velocity: 1.7
+    has_acceleration_limits: true
+    max_acceleration: 8.0
+  right_j1:
+    has_velocity_limits: true
+    max_velocity: 1.3
+    has_acceleration_limits: true
+    max_acceleration: 8.0
+  right_j2:
+    has_velocity_limits: true
+    max_velocity: 2.0
+    has_acceleration_limits: true
+    max_acceleration: 8.0
+  right_j3:
+    has_velocity_limits: true
+    max_velocity: 2.0
```

Comment:
Is this going to be an issue if this is slightly over the URDF limit of 1.957?

Reply:
Maybe. I updated the limits to be conservative across the board with respect to the velocity limits in the urdf.